src/main/java/config/DialogueConstants.java

```diff
@@ -14,6 +15,6 @@
     private static final Color borderColor = new Color(255, 0, 0, 255);
     private static final float padding = 3.f;
 
-    public static final JSONObject dialogueTree = JsonReader.readJson(Program.class.getResource("/JSON/dialogueTree.JSON").getPath());
+    public static final JSONObject dialogueTree = JsonReader.readJson(Objects.requireNonNull(Program.class.getResource("/JSON/dialogueTree.JSON")).getPath());
```

Comment:
This will create Errors in compiled JAR. Will this be fixed in another Branch?

Reply:
It did not, but i took your version anyway. ;-)